lib/std/heap.zig

```diff
@@ -8,6 +8,378 @@ const c = std.c;
 const Allocator = std.mem.Allocator;
 const windows = std.os.windows;
 
+const default_min_page_size: ?usize = switch (builtin.os.tag) {
+    .driverkit, .ios, .macos, .tvos, .visionos, .watchos => switch (builtin.cpu.arch) {
```

Comment:
`bridgeos` was added with the LLVM 19 upgrade in the meantime; should be added to all prongs like this.